src/main/java/Square.java

```diff
@@ -0,0 +1,30 @@
+public class Square extends Figure implements FigureInterface {
+    private double side;
+
+    public Square() {
+    }
+
+    public Square(String name) {
+        super(name);
+        this.side = Math.random();
```

Comment:
lets define random parameters of figures in FigureSupplier class, and accept them as constructor parameters